{"message":"Invalid session token","stack":"Error: Invalid session token\n    at Object.getAuthForSessionToken (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\Auth.js:114:11)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":209,"level":"error","timestamp":"2019-03-31T15:29:27.862Z"}
{"message":"Invalid session token","stack":"Error: Invalid session token\n    at Object.getAuthForSessionToken (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\Auth.js:114:11)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":209,"level":"error","timestamp":"2019-03-31T15:39:55.193Z"}
{"message":"Invalid session token","stack":"Error: Invalid session token\n    at Object.getAuthForSessionToken (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\Auth.js:114:11)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":209,"level":"error","timestamp":"2019-03-31T15:41:06.831Z"}
{"message":"Invalid session token","stack":"Error: Invalid session token\n    at Object.getAuthForSessionToken (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\Auth.js:114:11)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":209,"level":"error","timestamp":"2019-03-31T16:06:46.354Z"}
{"message":"Invalid session token","stack":"Error: Invalid session token\n    at Object.getAuthForSessionToken (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\Auth.js:114:11)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":209,"level":"error","timestamp":"2019-03-31T16:15:40.956Z"}
{"message":"Invalid session token","stack":"Error: Invalid session token\n    at Object.getAuthForSessionToken (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\Auth.js:114:11)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":209,"level":"error","timestamp":"2019-03-31T16:16:28.579Z"}
{"message":"Account already exists for this username.","stack":"Error: Account already exists for this username.\n    at config.database.find.then.results (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\RestWrite.js:494:13)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":202,"level":"error","timestamp":"2019-03-31T16:58:23.712Z"}
{"message":"Account already exists for this username.","stack":"Error: Account already exists for this username.\n    at config.database.find.then.results (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\RestWrite.js:494:13)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":202,"level":"error","timestamp":"2019-03-31T17:00:12.718Z"}
{"message":"Account already exists for this username.","stack":"Error: Account already exists for this username.\n    at config.database.find.then.results (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\RestWrite.js:494:13)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":202,"level":"error","timestamp":"2019-03-31T17:29:00.154Z"}
{"message":"Account already exists for this username.","stack":"Error: Account already exists for this username.\n    at config.database.find.then.results (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\RestWrite.js:494:13)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":202,"level":"error","timestamp":"2019-03-31T17:31:49.117Z"}
{"message":"Account already exists for this username.","stack":"Error: Account already exists for this username.\n    at config.database.find.then.results (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\RestWrite.js:494:13)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":202,"level":"error","timestamp":"2019-03-31T17:33:50.515Z"}
{"message":"schema mismatch for _User.Username; expected Object but got String","stack":"Error: schema mismatch for _User.Username; expected Object but got String\n    at reloadData.then (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\Controllers\\SchemaController.js:962:17)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":111,"level":"error","timestamp":"2019-03-31T18:13:46.645Z"}
{"message":"schema mismatch for _User.Username; expected Object but got String","stack":"Error: schema mismatch for _User.Username; expected Object but got String\n    at reloadData.then (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\Controllers\\SchemaController.js:962:17)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":111,"level":"error","timestamp":"2019-03-31T18:13:53.173Z"}
{"message":"schema mismatch for _User.Username; expected Object but got String","stack":"Error: schema mismatch for _User.Username; expected Object but got String\n    at reloadData.then (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\Controllers\\SchemaController.js:962:17)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":111,"level":"error","timestamp":"2019-03-31T18:14:17.131Z"}
{"message":"Account already exists for this username.","stack":"Error: Account already exists for this username.\n    at config.database.find.then.results (C:\\Users\\com\\parse-server\\node_modules\\parse-server\\lib\\RestWrite.js:494:13)\n    at process._tickCallback (internal/process/next_tick.js:68:7)","code":202,"level":"error","timestamp":"2019-03-31T18:33:05.671Z"}
